robolectric/src/test/java/org/robolectric/shadows/ShadowWindowManagerGlobalTest.java

```diff
@@ -22,6 +23,11 @@
 @RunWith(AndroidJUnit4.class)
 public class ShadowWindowManagerGlobalTest {
 
+  @Before
+  public void setup() {
+    System.setProperty("robolectric.areWindowsMarkedVisible", "true");
```

Comment:
I remember it is related to another commit in google branch or opening PR.

Reply:
Actually, before pushing it again i pulled the changes to my branch and then pushed..
That's why other changes appeared

Reply:
Let me fix it.